runtime/interp.c

```diff
@@ -218,7 +218,8 @@ sp is a local copy of the global variable Caml_state->extern_sp. */
 static __thread intnat caml_bcodcount;
 #endif
 
-static caml_root raise_unhandled;
+static value raise_unhandled;
+value global_data;
```

Comment:
I might have read it too fast, but is there a reason for `global_data` to be a C global?

Reply:
You're right, this need not be a C global. Will move its declaration to where it's used.